oaknational/Oak-Web-Application
# Fix/programmes static

## Description

make programmes page statically generated

## Issue(s)

Fixes #1622 

## How to test

1. Go to programmes 
2. Click on them
3. You should see pages aren't broken

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
